build.sbt

```diff
@@ -305,11 +305,13 @@ def nativeImageDockerSettings: Seq[Setting[_]] = dockerSettings ++ Seq(
   // If this is Some(…): run the native-image generation inside a Docker image
   // If this is None: run the native-image generation using a local GraalVM installation
   graalVMVersion := Some(GraalVersion),
-  graalVMNativeImageOptions ++= sharedNativeImageSettings({
-      graalVMVersion.value match {
-        case Some(_) => new File("/opt/docker/graal-resources/")
-        case None => baseDirectory.value / "src" / "graal"
-      }
+  graalVMNativeImageOptions ++= (graalVMVersion.value match {
+      case Some(_) =>
+        sharedNativeImageSettings(new File("/opt/docker/graal-resources/")) ++ Seq(
+          // disable build server for docker by default (and use 80% of available memory)
+          "--no-server"
```

Comment:
May I suggest we try to bake this into the sharedNativeImageSettings method—otherwise I think people might get confused if settings are added in multiple places.

Reply:
Okay, sure

Reply:
Updated to be a switch on the shared settings, and added an sbt setting for this so it can be set for local builds without needing to (un)comment.